lib/jsdom/living/events/MouseEvent.webidl

```diff
@@ -52,3 +51,27 @@ partial interface MouseEvent {
                         optional short buttonArg = 0,
                         optional EventTarget? relatedTargetArg = null);
 };
+
+// https://www.w3.org/TR/cssom-view/#extensions-to-the-mouseevent-interface
```

Comment:
Use https://drafts.csswg.org/cssom-view/ here and below; we don't use TR URLs.

Reply:
Done